README.md

```diff
@@ -1,12 +1,20 @@
-## Run locally
-
-1. `composer install`
-1. `cp .env.example .env` and configure `.env` file
-1. `yarn install`
-1. `yarn run dev`
-1. Optional: make the search work:
+## Local development
+
+1. Install [Valet](https://laravel.com/docs/9.x/valet)
+2. Install [sqlite](https://formulae.brew.sh/formula/sqlite)
+3. `composer install --no-progress --no-interaction --prefer-dist --optimize-autoloader`
+4. `yarn install --frozen-lockfile`
+5. `cp .env.example .env` and configure `.env` file
+   1. `php artisan key:generate --no-interaction` (generates APP_KEY)
+   2. `touch monica.db` and add path to DB_DATABASE
```

Comment:
IF we use sqlite, otherwise instructions are different. For instance, if they use Homestead.

Reply:
I'm not familiar with Homestead, can you draft them up. This is the one happy path I managed to explore.